.github/workflows/pull_request.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         uses: kiegroup/github-action-build-chain@v1.4
         with:
           parent-dependencies: 'optaweb-employee-rostering'
-          child-dependencies: 'kie-wb-distributions'
+          child-dependencies: 'kie-wb-distributions@7.x:master'
```

Comment:
how this works?

Reply:
@mareknovotny it is explained here https://github.com/marketplace/actions/github-action-build-chain-cross-repo-builds#input-fields
but as a summary it will map `optaweb-vehicle-routing:7.x` branch to `kie-wb-distributions:master` branch. The opposite is done here https://github.com/kiegroup/kie-wb-distributions/pull/1063/files#diff-4af9f3f60c8461140f4a9f265c8ff030R27